tests/components/mazda/test_device.py

```diff
@@ -0,0 +1,34 @@
+"""The device tests for the Mazda Connected Services integration."""
```

Comment:
Let's move these tests to `test_init.py` since that's where the source code is that we want to test.